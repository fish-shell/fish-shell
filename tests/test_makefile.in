source ../share/functions/__fish_print_make_targets.fish

# Setup
set workdir (mktemp -d)
cd $workdir
echo "
some\:target: # some:target
	echo "some:target"

some\:other\:target: ;

schmall: ; # schmall
" > Makefile


# Exercise
__fish_print_make_targets

# Teardown
rm -rf $workdir
