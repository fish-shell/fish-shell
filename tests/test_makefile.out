some:target
some:other:target
schmall
